dev/sql/00-base-schema.sql

```diff
@@ -4,8 +4,10 @@ CREATE TABLE `users` (
   `login` varchar(20) NOT NULL DEFAULT '',
   `password` varchar(40) NOT NULL DEFAULT '',
   `pw` varchar(255) NOT NULL DEFAULT '',
+  `about_me` varchar(1000) DEFAULT NULL,
```

Comment:
You don't want to change the schema here.

These schemas are numbered meaning they are meant to be executed one after another. In this case, you want to create a new file here to include your changes to the table so that your changes can be applied with ease with the existing database.

For instance:

```sql

ALTER TABLE `users` ADD COLUMN `about_me` varchar(1000) DEFAULT NULL;
ALTER TABLE `users` ADD COLUMN `is_verified` tinyint(1) DEFAULT NULL;
```